app/src/main/res/values-ru/strings.xml

```diff
@@ -122,7 +119,6 @@
     <string name="filter_close">"Закрыть"</string>
     <string name="filter_reset">"Сброс"</string>
     <string name="filter_apply">"Применить"</string>
-    <string name="filter_ssid_title">"SSID (Чувствительность к регистру символов)"</string>
```

Comment:
What is the reason to remove this? (It is translated)

Reply:
They were Google translated when you check the PR

I'm not looking for just coverage but quality